py-polars/src/error.rs

```diff
@@ -82,6 +82,7 @@ create_exception!(exceptions, SchemaFieldNotFoundError, PyException);
 create_exception!(exceptions, ShapeError, PyException);
 create_exception!(exceptions, StringCacheMismatchError, PyException);
 create_exception!(exceptions, StructFieldNotFoundError, PyException);
+create_exception!(exceptions, UnsupportedCPUError, PyException);
```

Comment:
I'm not sure this is the way to go. These exceptions are supposed to be raised in Python. Can't you just use a normal `Err` type instead of a PyException?

Reply:
@stinodego I'm confused, the exception *is* raised in Python. What do you mean by normal `Err` type?

Reply:
Now I am also confused 😄 

When would this error be thrown? I was thinking while building `py-polars` on an unsupported CPU, but I guess not?

Reply:
No, while loading a pre-compiled binary on unsupported machinery

Reply:
Right, in that case I think we must also add the exception to the module in `lib.rs` and import it in the Python package in the `exceptions.py` module.

Reply:
I do wonder if we can get there once this error occurs. A lot of our code will not work in python. This is also not an error you typically want to catch, as polars is useless at this point.

Reply:
Shouldn't we just throw a `PyImportError` then?

Reply:
@stinodego I would be fine with an `ImportError`.